ee/tabby-webserver/src/lib.rs

```diff
@@ -12,12 +12,13 @@ mod ui;
 pub mod public {
 
     pub use super::{
-        handler::attach_webserver,
+        handler::WebserverHandle,
         /* used by tabby workers (consumer of /hub api) */
         hub::api::{
             create_scheduler_client, create_worker_client, RegisterWorkerRequest, SchedulerClient,
             WorkerClient, WorkerKind,
         },
         /* used by examples/update-schema.rs */ schema::create_schema,
+        service::event_logger::new_event_logger,
```

Comment:
why this need to be public?

Reply:
Used in handler.rs, could make it `pub(crate)`

Reply:
handler should be able to reference `service::event_logger::new_event_logger` directly, as long as the function it self being marked as public (rather than re-export here)